scripts/test/Muon/workspace_finder_test.py

```diff
@@ -0,0 +1,75 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+
+from mantid.py3compat import mock
+from mantidqt.utils.qt.testing import start_qapplication
+from Muon.GUI.Common.test_helpers.context_setup import setup_context
+from Muon.GUI.Common.plotting_widget.workspace_finder import WorkspaceFinder
+from Muon.GUI.Common.muon_group import MuonGroup
+
+
+@start_qapplication
+class WorkspaceFinderTest(unittest.TestCase):
+    def setUp(self):
+        self.context = setup_context(True)
+        self.workspace_finder = WorkspaceFinder(self.context)
+
+        test_group = MuonGroup(group_name="fwd", detector_ids=[1, 2, 3, 4, 5])
+        self.context.group_pair_context._groups = [test_group]
+        self.context.group_pair_context._pairs = []
+        self.context.group_pair_context._selected_groups = ["fwd"]
+        self.context.group_pair_context._selected_pairs = []
+
+    def test_get_workspace_to_plot_calls_correctly_if_frequency_domain(self):
+        self.workspace_finder.get_freq_workspaces_to_plot = mock.Mock(return_value=["62260;fwd"])
+        self.workspace_finder.get_time_workspaces_to_plot = mock.Mock(return_value=["62260;fwd"])
+
+        self.workspace_finder.get_workspaces_to_plot(True, "Frequency Re")
+        self.assertEquals(self.workspace_finder.get_freq_workspaces_to_plot.call_count, 1)
+        self.assertEquals(self.workspace_finder.get_time_workspaces_to_plot.call_count, 0)
+
+    def test_get_workspace_to_plot_calls_correctly_if_time_domain(self):
+        self.workspace_finder.get_freq_workspaces_to_plot = mock.Mock(return_value=["62260;fwd"])
+        self.workspace_finder.get_time_workspaces_to_plot = mock.Mock(return_value=["62260;fwd"])
+
+        self.workspace_finder.get_workspaces_to_plot(True, "Asymmetry")
+
+        self.assertEquals(self.workspace_finder.get_freq_workspaces_to_plot.call_count, 0)
+        self.assertEquals(self.workspace_finder.get_time_workspaces_to_plot.call_count, 1)
+
+    def test_get_workspace_to_plot_time(self):
+        runs = [[62260]]
+        self.context.data_context.current_runs = runs
```

Comment:
You can set current runs directly, no need to use a runs variable.

Reply:
I was favoring a variable, so I didn't have to rewrite the same information [[62260]] in the assert calls.